mass_editing/models/mass_object.py

```diff
@@ -10,7 +10,7 @@ class MassObject(models.Model):
     _name = "mass.object"
     _description = "Mass Editing Object"
 
-    name = fields.Char('Name', required=True, select=1)
+    name = fields.Char('Name', required=True, index=1)
```

Comment:
FYI we have developed a pylint check for this case.
I just enabled from 
https://github.com/Vauxoo/pylint-conf/commit/3a63d5917ddf6b85f82022fb80605dd84ed9a734

cc @JesusZapata